### PR TITLE
Change Navigation Item alignment and update docs/examples

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
- Change `NavigationItem` flex alignment so it works well with longer labels
- Rename `blurSelected` prop to `blurActive`
- Add new `--salt-size-indicator` to `--navigationItem-bar-size` css var
- Update description on `blurSelected` prop
- Move `ConditionalWrapper` outside `NavigationItem`
- Add longer label example to storybook
- Add `FlexLayout` to storybook examples
- Update docs following content review

### CODE REVIEW

**Review by origami-z — COMMENTED** [review]
When using keyboard 'space' to toggle group expansion, should focus remain on the parent item given no navigation would happen?

**Comment by JoanaMMoreira** [comment]
> When using keyboard 'space' to toggle group expansion, should focus remain on the parent item given no navigation would happen?

Good point, I'll amend it

File: packages/lab/src/navigation-item/NavigationItem.tsx [hunk 1]
```diff
@@ -89,13 +91,20 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
       onExpand?.(event);
     };
 
+    const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+    useEffect(() => {
+      expanded && buttonRef.current?.focus();
```
Comment (origami-z) [inline packages/lab/src/navigation-item/NavigationItem.tsx:97]: I don't think this is the solution to focus being lost.

It's very likely due to `ConditionalWrapper` where a new Button is created. Can try refactor the code so `Button` stays the same

Comment (JoanaMMoreira) [reply packages/lab/src/navigation-item/NavigationItem.tsx]: Should be fixed now 😄

**Review by Fercas123 — DISMISSED** [review]

File: packages/lab/src/navigation-item/NavigationItem.tsx [hunk 2]
```diff
@@ -50,9 +50,48 @@ export interface NavigationItemProps extends ComponentPropsWithoutRef<"div"> {
...
+interface ConditionalWrapperProps
+  extends Pick<
+    NavigationItemProps,
+    "parent" | "expanded" | "onExpand" | "active" | "href"
+  > {
   children: ReactNode;
   className: string;
+}
+
+const ConditionalWrapper = ({
```
Comment (alycrys) [inline packages/lab/src/navigation-item/NavigationItem.tsx:62]: would it make sense to have it in a separate file?

File: .changeset/empty-tips-thank.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+"@salt-ds/lab": patch
+---
+
+- Change `NavigationItem` flex alignment so it works well with longer labels
+- Add new `--salt-size-indicator` to `--navigationItem-bar-size` css var
```
Comment (alycrys) [inline .changeset/empty-tips-thank.md:6]: don't think we need to mention the css changes. Or do we?

**Review by alycrys — DISMISSED** [review]

File: .changeset/empty-tips-thank.md (hunk 3 above)
Comment (JoanaMMoreira) [reply .changeset/empty-tips-thank.md:6]: probably not, I'll remove this

**Review by alycrys — APPROVED** [review]

**Pull request merged**